### PR TITLE
Major rewrite

### PR DESCRIPTION
**Pull request by 73cirdan**
updated the module a written in the readme
redid the lint part (prettier, eslint,stylelint), added html prettier plugin
the workflow part is not in my code, leave it untouched at your side?
the unittest part did not work on my side, was it working for you ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +43,7 @@ cd ~/MagicMirror/modules
 Clone this repository:
 
 ```bash
-git clone https://github.com/ianperrin/MMM-Formula1
+git clone https://github.com/73cirdan/MMM-Formula1
```
Comment (KristjanESPERANTO) [inline README.md:46]: This should be undone before the PR is accepted.

File: package.json [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
-		"stylelint": "^15.3.0",
-		"stylelint-config-standard": "^31.0.0",
-		"stylelint-prettier": "^3.0.0"
-	},
-	"dependencies": {
-		"f1-api": "^1.2.0",
-		"ical-generator": "^3.6.1"
-	}
+  "name": "mmm-formula1",
+  "version": "0.8.0",
```
Comment (KristjanESPERANTO) [inline package.json:3]: Why do you want to change it from 1.4.0 to 0.8.0?

Comment (73cirdan) [reply package.json:3]: not sure how that happened. We discussed about setting it to 2.0.0 in the release to come

**Comment by ianperrin** [comment]
Yes - support setting the version to 2.0.0 and updating the readme so it references this ***@***.*** can you update the pull request?On 29 Mar 2024, at 20:18, cirdan ***@***.***> wrote:﻿
@73cirdan commented on this pull request.



In package.json:
> -		"eslint-config-prettier": "^8.7.0",
-		"eslint-plugin-jsdoc": "^40.0.3",
-		"eslint-plugin-prettier": "^4.2.1",
-		"mocha": "^10.2.0",
-		"module-alias": "^2.2.2",
-		"prettier": "^2.8.4",
-		"stylelint": "^15.3.0",
-		"stylelint-config-standard": "^31.0.0",
-		"stylelint-prettier": "^3.0.0"
-	},
-	"dependencies": {
-		"f1-api": "^1.2.0",
-		"ical-generator": "^3.6.1"
-	}
+  "name": "mmm-formula1",
+  "version": "0.8.0",

not sure how that happened. We discussed about setting it to 2.0.0 in the release to come

File: README.md (hunk 1 above)
Comment (73cirdan) [reply README.md:46]: updated package.json to ianperrin

File: package.json (hunk 2 above)
Comment (73cirdan) [reply package.json]: updated version in package.json

**Pull request merged**